dozer-types/src/types/mod.rs

```diff
@@ -238,3 +242,103 @@ pub enum Operation {
     Insert { new: Record },
     Update { old: Record, new: Record },
 }
+
+#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
+pub struct DozerCoord(pub Coord);
+
+impl Eq for DozerCoord {}
```

Comment:
Instead of implementing ordering traits, we can do this:

```rust
#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
pub struct OrderedCoordFloat(pub OrderedFloat<f64>);

impl geo::CoordFloat for OrderedCoordFloat {}

#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
pub struct DozerCoord(pub Coord<OrderedCoordFloat>);
```

Reply:
I have removed `DozerCoord`, which your suggestion was referring to.

Anyway, I have tried to use a similar approach for `DozerPoint`, but to use OderedFloat, we need to implement a bunch of methods from supertrait of `CoordFloat`.

Reply:
I'm afraid that's necessary. Implementing `Eq` for `f64` is unsound.